doc/user_guide/transformations.rst

```diff
@@ -628,7 +639,7 @@ A valid script file must contain a **trans** function which accepts a **PSy**
 object as an argument and returns a **PSy** object, i.e.:
 ::
 
-    def trans(psy)
+    def trans(psy):
```

Comment:
Done. I also updated the other snippets too. This includes the interactive example for which I had to create an input file and check the various outputs. If you think the interactive example is now overkill and less readable then let me know and I'll revert. Note the last part of the interactive example only has one ">". That is to avoid using doctest as it fails due to the output from view() including colour sequences. That will be fixed with an option in the future, see #1528.